src/components/NcCollectionList/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+export { default as NcCollectionListItem } from './NcCollectionListItem.vue'
```

Comment:
Could you split this into two separate components?
Meaning `src/components/NcCollectionList` and `src/components/NcCollectionListItem`

This would match our other components

Reply:
Sorry, didn't get you there. I used structure from NcRichContenteditable as an example, is that not the same?

Anyway, `ListItem` is internal component, and shouldn't be used separately from `List`, so maybe we don't need to re-export it (like `NcUserBubbleDiv`)?

Reply:
It depends I would say, if it is private do not export it then it could be like it is right now.